pkg/cmd/util/clientcmd/clientconfig.go

```diff
@@ -18,6 +18,10 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 	overrideFlags.ContextOverrideFlags.Namespace.ShortName = "n"
 	overrideFlags.AuthOverrideFlags.Username.LongName = ""
 	overrideFlags.AuthOverrideFlags.Password.LongName = ""
+
+	// for compatibility with kubernetes, it is still provided
+	overrideFlags.ClusterOverrideFlags.APIServer.ShortName = "s"
```

Comment:
Can you add a test to make sure our default bindings don't slip from kube's default bindings?